README.md

```diff
@@ -17,7 +17,10 @@ save the anything to the database.
 
 Commands are automatically picked up from config files stored within:
 
-`/var/lib/adminware/tools/{batch,open}/<command-name>/config.yaml`
+`/var/lib/adminware/tools/{batch,open}/[optional-other-directories/]<command-name>/config.yaml`
```

Comment:
As the `namespace` can repeat multiple times, can you update the text to : `[<optional-namespace>/].../<command-name>`

Had to check the standard for the repetition: http://pubs.opengroup.org/onlinepubs/9699919799/basedefs/V1_chap12.html

Reply:
done here https://github.com/alces-software/adminware/pull/87/commits/c905fed4fdeaca7e9592c60033e2ad11aa3e96ba